src/custom_nodes/tokenizer/README.md

```diff
@@ -85,7 +85,7 @@ The outputs of this node match inputs of GPT-like models.
 OVMS supports automatic conversion from `U8 2D` format to TensorflowServing/KServe proto.
 
 # Custom node parameters
-Parameters can be defined in pipeline definition in OVMS configuration file. [Read more](https://github.com/openvinotoolkit/model_server/blob/main/docs/custom_node_development.md) about node parameters.
+Parameters can be defined in pipeline definition in OVMS configuration file. [Read more](https://github.com/openvinotoolkit/model_server/blob/releases/2024/1/ses/2024/1/docs/custom_node_development.md) about node parameters.
```

Comment:
this one looks weird, are you sure it's okay?

Reply:
Its in fact wrong, pushed fix